app/components/dex-indicator.jsx

```diff
@@ -0,0 +1,20 @@
+import { connect } from 'react-redux';
+
+export function DexIndicator ({ dex, profile }) {
+
+  if ((dex && !dex.shiny) || profile) {
```

Comment:
hmm is this `|| profile` necessary? cause wont `dex` be `undefined` if `profile` is true?